content/config-dev.yml

```diff
@@ -1,4 +1,3 @@
 ---
 output_folder: ../dev-output
 mode: dev
-google_analytics_id: 'UA-36510846-2'
```

Comment:
Hm, actually I wasn't sure why we're even doing analytics on local development, and I was planning to remove this. (We have a different GA ID for production set in `config-prod.yml`.) @SondraE, are you actually using the "OddSite dev" analytics app for anything? Any reason to keep it around?